README.md

```diff
@@ -3,17 +3,28 @@
 
 ## Requirements
 
-- Node.JS ~6.2
-- npm ~3.9
+- Node.JS ~6.10
+- npm ~3.10
+- Yarn ~0.23.2
 - Docker (optional)
 
-## Getting started
+## Documentation
+
+Article converter is an api for getting a extended html version of the content attribute provided by [Article API](https://github.com/NDLANO/article-api). The service has two endpoints:
+
+**/article-converter/json/:lang/:articleId**
+
+_Returns an extended and transformed json structure based on the one provided by [Article API](https://github.com/NDLANO/article-api)_
+
+**/article-converter/html/:lang/:articleId**
+
+_Returns the content attribute from [Article API](https://github.com/NDLANO/article-api) transformed to plain html and wrapped in a HTML document (useful for testing)_
 
-What's in the box?
+The service mainly converts `<embed>` tags in the content attribute to appropriate html tags. For some embed tags fetching additional data from other api's is required.
 
-- Express
-- React
-- Babel (ES6)
+*N.B.* To properly display the converted html some script from [ndla-article-scripts](https://github.com/NDLANO/frontend-packages/tree/master/packages/ndla-article-scripts) is required. See [ndla-frontend](https://github.com/NDLANO/ndla-frontend) for examples.
+
+## Getting started
```

Comment:
Burde dette renavnes til `Developer documentation` el.n.sånt? For å skille bedre mot det funksjonelle ovenfor...